include/torch-mlir/Dialect/Torch/IR/TorchOps.h

```diff
@@ -267,6 +268,41 @@ m_TorchTensorSizeInt(Value tensor, int64_t *dim) {
   return detail::torch_tensor_size_int_op_binder(tensor, dim);
 }
 
+namespace detail {
+/// Matches the constant integers stored in a `onnx.Constant`.
+struct onnx_list_of_constant_ints_op_binder {
```

Comment:
This should not be in `TorchOps.h`. You can move it into the `Utils.h` file for `onnx`.